.pre-commit-config.yaml

```diff
@@ -1,17 +1,23 @@
+---
 repos:
-  - repo: 'https://github.com/pre-commit/pre-commit-hooks'
-    rev: v4.5.0
-    hooks:
-      - id: check-yaml
-      - id: end-of-file-fixer
-      - id: trailing-whitespace
-  - repo: 'https://github.com/PyCQA/isort'
-    rev: 5.12.0
-    hooks:
-      - id: isort
-  - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.4.8
-    hooks:
-      - id: ruff
-        args: [ --fix ]
-      - id: ruff-format
+- repo: 'https://github.com/pre-commit/pre-commit-hooks'
+  rev: v4.5.0
+  hooks:
+  - id: check-yaml
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+- repo: 'https://github.com/PyCQA/isort'
+  rev: 5.12.0
+  hooks:
+  - id: isort
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.4.8
+  hooks:
+  - id: ruff
+    args: [--fix]
```

Comment:
For the future: formatters should go before linters checking the same files. I also soft the checks by speed where possible.

Reply:
Oh, I didn't realize that you inherit linters from them..

Reply:
Yes, also several of the dev dependency pins, such as `pytest`.